### PR TITLE
[Virtual-Wan] migrate to track2

### PR DESCRIPTION
**Pull request by 00Kai0**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Virtual wan on Track 2

File: src/virtual-wan/azext_vwan/custom.py [hunk 1]
```diff
@@ -334,16 +361,16 @@ def update_vhub_route_table(cmd, resource_group_name, virtual_hub_name, route_ta
...
 
 
 def get_vhub_route_table(cmd, resource_group_name, virtual_hub_name, route_table_name):
-    from msrestazure.azure_exceptions import CloudError
+    from azure.core.exceptions import ResourceNotFoundError
     try:
         return _v3_route_table_client(cmd.cli_ctx)\
             .get(resource_group_name, virtual_hub_name, route_table_name)  # Get v3 route table first.
-    except CloudError as ex:
+    except ResourceNotFoundError as ex:
```
Comment (jsntcy) [inline src/virtual-wan/azext_vwan/custom.py:373]: >ResourceNotFoundError [](http://example.com/codeflow?start=11&length=21)

`ResourceNotFoundError` already means `ex.status_code == 404`.

The original logic is we won't raise exception only when status_code is 404; other cases we should raise exception.

Use following code is enough,

```Python
except ResourceNotFoundError as ex:
            return _v2_route_table_client(cmd.cli_ctx)\
                .get(resource_group_name, virtual_hub_name, route_table_name)  # Get v2 route table.
```

**Comment by jsntcy** [comment]
@00Kai0, BTW, don't forget upgrade extension version and add release notes.

**Review by jsntcy — APPROVED** [review]
:shipit:

**Review by jsntcy — APPROVED** [review]

**Pull request merged**